packages/ui/src/composables/useBreakpoint.ts

```diff
@@ -15,20 +22,25 @@ const defaultHelpers = helpersKeys.reduce((acc, key) => {
 export const useBreakpoint = (): BreakpointServiceObject => {
   const injected = inject(vaBreakpointSymbol, {}) as BreakpointServiceObject
 
-  const globalConfig = useGlobalConfig().globalConfig.value.breakpoint
+  const isMounted = useIsMounted()
 
-  const defaultBreakpoint = globalConfig.enabled
+  const { globalConfig } = useGlobalConfig()
+
+  const breakpointConfig: ComputedRef<BreakpointConfig> = computed(() => {
+    const breakpoint = globalConfig.value.breakpoint
+    if (!breakpoint) { warn('useBreakpoint: breakpointConfig is not defined!') }
+    return breakpoint ?? {} as BreakpointConfig
+  })
+
+  const defaultBreakpoint = breakpointConfig.value.enabled
     ? {
       width: undefined,
       height: undefined,
       current: undefined,
-      thresholds: globalConfig.thresholds,
+      thresholds: breakpointConfig.value.thresholds,
       ...defaultHelpers,
     }
     : {} as BreakpointServiceObject
 
-  const isMounted = ref(false)
-  onMounted(() => { isMounted.value = true })
-
   return useReactiveComputed(() => isMounted.value ? injected : defaultBreakpoint)
```

Comment:
You still losing reactivity here, because defaultBreakpoint is not a ref (or computed). 
```suggestion
  return useReactiveComputed(() => isMounted.value ? injected : defaultBreakpoint.value)
```